raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -47,7 +48,7 @@ contract NettingChannelContract {
         (success, balance) = data.deposit(amount);
 
         if (success == true) {
-            ChannelNewBalance(data.token, msg.sender, balance, data.opened);
+            ChannelNewBalance(data.token, msg.sender, balance, block.number);
```

Comment:
Because it allows for _slightly-off_ errors. The current change allows to use the number from the event with no drawback (other than deprecation). Another option I could see is to hardcode it to `0` -- that way no one can use it inadvertently.